pkg/trie/triedb/triedb_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package triedb
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/pkg/trie"
+	"github.com/ChainSafe/gossamer/pkg/trie/triedb/db"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInsertions(t *testing.T) {
+	inmemoryDB := db.NewMemoryDB(make([]byte, 1))
+	trieDB := NewTrieDB(trie.EmptyHash, inmemoryDB, nil)
+
+	entries := map[string][]byte{
+		"no":        []byte("no"),
```

Comment:
Perhaps here (even though it is string vs []byte) we can make the string turned into byes slightly different than the associated Key? Something like `"no":        []byte("noVal")`

Reply:
Done